JS/models/MilitaryType.js

```diff
@@ -0,0 +1,7 @@
+const MilitaryType = {
```

Comment:
Please use CAPITAL_LETTER syntax for constants.

Reply:
Done